components/hacbs/jvm-build-service/base/kustomization.yaml

```diff
@@ -1,5 +1,6 @@
 resources:
 - https://github.com/redhat-appstudio/jvm-build-service/deploy/kcp?ref=954c86b39a12820c7910c93b1e20b9b572232816
+- self-binding.yaml
```

Comment:
let's change the commit ^^ to `2bda08e7d8ff0b3d78c6564f54f183208b44af18` to pull in fixes from https://github.com/redhat-appstudio/jvm-build-service/pull/270

Reply:
or should we replicate https://github.com/redhat-appstudio/build-service/pull/65 in jvm-build-service first and then move to that commit ?

Reply:
going to update jvm-build-service to update automatically.